src/File/Params/UploadFileParameters.php

```diff
@@ -83,6 +83,18 @@ public function setLocalesToApprove($locales_to_approve)
         return $this;
     }
 
+    /**
+     * @param string $namespace
+     *
+     * @return $this
+     */
+    public function setNamespace($namespace)
+    {
+      $this->set('smartling.namespace', $namespace);
```

Comment:
Pls add validation for null\empty string (+unit test), otherwise you will get error
```json
{
    "response": {
        "code": "GENERAL_ERROR",
        "errors": [
            {
                "key": "parse.error",
                "message": "There was a problem loading your file. java.lang.IllegalArgumentException: Not empty argument 'namespaceName' is expected here.",
                "details": {
                    "errorId": "irou01yo"
                }
            }
        ]
    }
}
```

Reply:
Done, please validate